metricbeat/module/kubernetes/state_replicaset/data.go

```diff
@@ -0,0 +1,59 @@
+package state_replicaset
```

Comment:
Yeah, I'm not 100% happy with this either, we can see how it goes and perhaps improve it in the future